### DISCUSSION TITLE
Download old version of ImageMagick (7.0.10-16)

### DISCUSSION THREAD

**Discussion by JanC89** [ImageMagick/ImageMagick#3785] [Help]
Hi,

Long story short, we have a service running which requires (among other) ImageMagick to be installed on the host. 
The server is being configured using Ansible, and the build file currently downloads:

- https://imagemagick.org/download/linux/CentOS/x86_64/ImageMagick-libs-7.0.10-16.x86_64.rpm
- https://imagemagick.org/download/linux/CentOS/x86_64/ImageMagick-7.0.10-16.x86_64.rpm

These files no longer exist so ansible fails. The obvious solution would be to just install the newest version, but due to various dependency problems on the server, that is not feasible. 

So, I was wondering if someone had a link to the above rpm files so I could download them and store them locally?

**Comment by urban-warrior** [comment]
You can find most releases of ImageMagick here: https://download.imagemagick.org/ImageMagick/download/releases/.
